### PR TITLE
Release 0.1.5

### PR DESCRIPTION
**Pull request by ndr-brt**
Carry 0.1.5 release changes from `product-edc`.

Details:
https://github.com/catenax-ng/product-edc/releases/tag/0.1.5

### CODE REVIEW

**Comment by carslen** [comment]
Suggestion for future pull requests:
- split your PRs into smaller peaces
  - e.g. one PR per feature
  - separate changes in workflows, documentation, code (maybe even between controll-plane and data-plane) into seperate PRs

Please do a content review and approve the programatic part of the PR here in Eclipse-Tractusx repository. Even if you are not a committer yet, you are able to review/approve. This will also count as contribution for possible Eclipse Committer elections. After your review assign the PR to an Eclipse Committer. Refer also to our brand new [review guide](https://eclipse-tractusx.github.io/docs/oss/code-reviews).

This will make it easier for us to check and approve/merge PRs without detailed knowledge of the product itself.

File: charts/edc-dataplane/Chart.yaml [hunk 1]
```diff
@@ -3,8 +3,8 @@ apiVersion: v2
 name: edc-dataplane
 description: >-
   EDC Data-Plane - The Eclipse DataSpaceConnector data layer with responsibility of transferring and receiving data streams
-home: https://github.com/eclipse-tractusx/tractusx-edc
+home: https://github.com/catenax-ng/product-edc/charts/edc-dataplane
```
Comment (carslen) [inline charts/edc-dataplane/Chart.yaml:6]: Do not use NG organization in Eclipse-Tractusx. The home of the source code is the Eclipse-Tractusx repository, not the repo in the NG org.

Please add 
- License header to (all) yaml files
- key `source` to this file (refer to [TRG 5.02](https://eclipse-tractusx.github.io/docs/release/trg-5/trg-5-2#the-chartyaml-file))

File: charts/edc-dataplane/README.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ EDC Data-Plane - The Eclipse DataSpaceConnector data layer with responsibility o
 ## TL;DR
 ```shell
 $ helm repo add catenax-ng-product-edc https://catenax-ng.github.io/product-edc
-$ helm install my-release catenax-ng-product-edc/edc-dataplane --version 0.1.2
+$ helm install my-release catenax-ng-product-edc/edc-dataplane --version 0.1.5
```
Comment (carslen) [inline charts/edc-dataplane/README.md:12]: Do not use references to catenax-ng GH organization in Eclipse-Tractusx. For your example, you can use our central Helm repository available at https://eclipse-tractusx.github.io/charts/dev:

```
$ helm repo list
NAME        	URL
tractusx-dev	https://eclipse-tractusx.github.io/charts/dev
$ helm search repo edc
NAME                         	CHART VERSION	APP VERSION	DESCRIPTION
tractusx-dev/edc-controlplane	0.1.2        	0.1.2      	EDC Control-Plane - The Eclipse DataSpaceConnec...
tractusx-dev/edc-dataplane   	0.1.2        	0.1.2      	EDC Data-Plane - The Eclipse DataSpaceConnector...
```

File: edc-controlplane/edc-controlplane-postgresql-hashicorp-vault/src/main/docker/Dockerfile [hunk 3]
```diff
@@ -10,13 +10,14 @@
...
-FROM alpine:3.16.2 as otel
+
+FROM alpine:3.17.1 as otel
 
 ENV OTEL_AGENT_LOCATION "https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/download/v1.12.1/opentelemetry-javaagent.jar"
 
 RUN wget ${OTEL_AGENT_LOCATION} -O /tmp/opentelemetry-javaagent.jar
 
-FROM alpine:3.16.2
+FROM alpine:3.17.1
```
Comment (carslen) [inline edc-controlplane/edc-controlplane-postgresql-hashicorp-vault/src/main/docker/Dockerfile:20]: For Java based applications please use [Eclipse Temurin](https://hub.docker.com/_/eclipse-temurin) images as base image ([TRG 4.02](https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-2)).

File: edc-controlplane/edc-controlplane-postgresql/src/main/docker/Dockerfile [hunk 4]
```diff
@@ -10,13 +10,14 @@
 #  Contributors:
 #       Mercedes-Benz Tech Innovation GmbH - Initial Dockerfile
 #
-FROM alpine:3.16.2 as otel
+
```
Comment (carslen) [inline edc-controlplane/edc-controlplane-postgresql/src/main/docker/Dockerfile:13]: License header not valid.

File: edc-controlplane/edc-controlplane-postgresql/src/main/docker/Dockerfile [hunk 5]
```diff
@@ -10,13 +10,14 @@
...
+FROM alpine:3.17.1 as otel
 
 ENV OTEL_AGENT_LOCATION "https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/download/v1.12.1/opentelemetry-javaagent.jar"
 
 RUN wget ${OTEL_AGENT_LOCATION} -O /tmp/opentelemetry-javaagent.jar
 
-FROM alpine:3.16.2
+FROM alpine:3.17.1
```
Comment (carslen) [inline edc-controlplane/edc-controlplane-postgresql/src/main/docker/Dockerfile:20]: For Java based applications please use [Eclipse Temurin](https://hub.docker.com/_/eclipse-temurin) images as base image ([TRG 4.02](https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-2)).

File: edc-dataplane/edc-dataplane-azure-vault/src/main/docker/Dockerfile [hunk 6]
```diff
@@ -10,13 +10,13 @@
 #  Contributors:
```
Comment (carslen) [inline edc-dataplane/edc-dataplane-azure-vault/src/main/docker/Dockerfile:10]: License header again.

File: edc-dataplane/edc-dataplane-azure-vault/src/main/docker/Dockerfile [hunk 7]
```diff
@@ -10,13 +10,13 @@
...
 #
-FROM alpine:3.16.2 as otel
+FROM alpine:3.17.1 as otel
 
 ENV OTEL_AGENT_LOCATION "https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/download/v1.12.1/opentelemetry-javaagent.jar"
 
 RUN wget ${OTEL_AGENT_LOCATION} -O /tmp/opentelemetry-javaagent.jar
 
-FROM alpine:3.16.2
+FROM alpine:3.17.1
```
Comment (carslen) [inline edc-dataplane/edc-dataplane-azure-vault/src/main/docker/Dockerfile:19]: Base image again

File: edc-dataplane/edc-dataplane-hashicorp-vault/src/main/docker/Dockerfile [hunk 8]
```diff
@@ -10,13 +10,13 @@
 #  Contributors:
 #       Mercedes-Benz Tech Innovation GmbH - Initial Dockerfile
 #
```
Comment (carslen) [inline edc-dataplane/edc-dataplane-hashicorp-vault/src/main/docker/Dockerfile:12]: License header and base image again.

**Review by carslen — CHANGES_REQUESTED** [review]
See comments and please check your code and use correct license header.
